Packs/CybleEventsV2/Integrations/CybleEventsV2/CybleEventsV2.py

```diff
@@ -94,15 +288,213 @@ def test_response(client, method, base_url, token):
+        original_datetime = datetime.strptime(timestamp_str, "%Y-%m-%dT%H:%M:%S.%fZ")
+        updated_datetime = original_datetime + timedelta(microseconds=1000)
+        latest_created_time = updated_datetime.strftime("%Y-%m-%dT%H:%M:%S.%fZ")
+
+        events = format_incidents(alerts, hide_cvv_expiry)
+
+        for event in events:
+
+            inci = {
+                'name': event.get('name'),
+                'severity': event.get('severity'),
+                'rawJSON': json.dumps(event),
+                'alert_group_id': event.get('alert_group_id'),
+                'event_id': event.get('event_id'),
+                'keyword': event.get('keyword'),
+                'created': event.get('created_at')
+            }
+            incidents.append(inci)
+
+        next_run = {'event_pull_start_date': latest_created_time}
+
+        return incidents, next_run
+    else:
+        return [], {'event_pull_start_date': latest_created_time}
+
+
+def update_remote_system(client, method, token, args, url):
+
+    parsed_args = UpdateRemoteSystemArgs(args)
+
+    if parsed_args.delta:
+
+        severities = {
+            "1": "LOW",
+            "2": "MEDIUM",
+            "3": "HIGH",
+            "4": "CRITICAL"
+        }
+
+        data = parsed_args.data
+
+        incident_id = data.get('id')
+        status = data.get('status')
+        assignee_id = data.get('assignee_id')
+        updated_severity = str(data.get('severity'))
+
+        updated_event = {
+            "id": incident_id
+        }
+
+        if status in INCIDENT_STATUS:
+            updated_event["status"] = status
+
+        if assignee_id:
+            updated_event["assignee_id"] = assignee_id
+
+        if updated_severity:
+            if updated_severity == "0.5" or updated_severity == "0":
+                updated_event["user_severity"] = None
+            else:
+                updated_event["user_severity"] = severities.get(updated_severity)
+
+        body = {
+            "alerts": [updated_event]
+        }
+
+        set_request(client, method, token, body, url)
+
+    return None
+
+
+def get_mapping_fields(client, token):
+
+    input_params = {}
+
+    input_params['order_by'] = "asc"
+    input_params['from_da'] = 0
+    input_params['limit'] = 500
+
+    initial_interval = 1
+
+    event_pull_start_date = datetime.utcnow() - timedelta(days=int(initial_interval))
+
+    input_params['start_date'] = event_pull_start_date.astimezone().isoformat()
+    input_params['end_date'] = datetime.utcnow().astimezone().isoformat()
+
+    final_input_structure = alert_input_structure(input_params)
 
-    """
-    fetch = fetch_subscribed_services(client, method, base_url, token)
-    if fetch:
-        return 'ok'
-    else:
-        demisto.error("Failed to connect")
-        raise Exception("failed to connect")
+    alerts = set_request(client, 'POST', token, final_input_structure, 'https://api.cyble.ai/apollo/api/v1/y/alerts')
```

Comment:
I think the address should be as a constant at the top of the code, that way if it's changed it will be easy to find.

Reply:
Yes, fixed it.